packages/core/src/fhir-deduplication/__tests__/group-same-medications.test.ts

```diff
@@ -64,18 +64,14 @@ describe("groupSameMedications", () => {
     expect(snomedMap.size).toBe(0);
   });
 
-  it("does not lose medications that have neither snomed nor icd-10 codes", () => {
+  it("loses medications that have neither of the expected codes", () => {
```

Comment:
this test is saying we lose medications that dont have an expected code system?

Reply:
to clarify, can we also have a test that shows that the removeOtherCodes function works? I was a bit confused since there are two different distinct behaviors here:

1. dont put medications into the map that dont have either rxnorm or ndc snomed
2. for medications that do have valid system but also have epic or other shitty code system, strip those codes from the medication

Reply:
added 2)